backend/src/test/groovy/pt/ulisboa/tecnico/socialsoftware/tutor/submission/CreateReviewPerformanceTest.groovy

```diff
@@ -0,0 +1,60 @@
+package pt.ulisboa.tecnico.socialsoftware.tutor.submission.service
+
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.context.TestConfiguration
+import pt.ulisboa.tecnico.socialsoftware.tutor.question.domain.Question
+import pt.ulisboa.tecnico.socialsoftware.tutor.submission.domain.Submission
+import pt.ulisboa.tecnico.socialsoftware.tutor.submission.dto.ReviewDto
+import pt.ulisboa.tecnico.socialsoftware.tutor.SpockTest
+import pt.ulisboa.tecnico.socialsoftware.tutor.BeanConfiguration
+import pt.ulisboa.tecnico.socialsoftware.tutor.user.User
+
+@DataJpaTest
+class CreateReviewPerformanceTest extends SpockTest{
+    def student
```

Comment:
not necessary to have  a performance test until we define scenario, it is nor expectd to have 100's of users doing reviews